src/Middleware/RequestThrottling/sample/Startup.cs

```diff
@@ -14,40 +14,41 @@ namespace RequestThrottlingSample
 {
     public class Startup
     {
+        static IConfiguration _config;
+
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddTailDropQueue((options) =>
             {
-                options.MaxConcurrentRequests = 4;
-                options.RequestQueueLimit = 0;
+                options.MaxConcurrentRequests = Math.Max(1, _config.GetValue<int>("maxCores"));
+                options.RequestQueueLimit = Math.Max(1, _config.GetValue<int>("maxQueue"));
             });
 
-            services.AddLogging();
+            //services.AddLogging();
```

Comment:
Remove or uncomment.

Reply:
I had made several changes to the sample project, since I was running it on the benchmark server. Now that I'm adding real scenarios to the benchmarks repo, I reverted those changes here.